javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1623,6 +1623,22 @@ class WebDriver {
     }
   }
 
+  async createSandbox(frameId, name = null, grantUniversalAccess = false) {
+    let connection
+    if (Object.is(this._cdpConnection, undefined)) {
+      connection = await this.createCDPConnection('page')
+    } else {
+      connection = this._cdpConnection
+    }
+
+    return await connection.send(
+      'Page.createIsolatedWorld',
+      frameId,
+      name,
+      grantUniversalAccess
+    )
```

Comment:
This should be passed as an object. `name` should be changed to `worldName` while sending as an object as per the CDP docs here: https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-createIsolatedWorld